pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -529,7 +529,7 @@ func (a *Adapter) createOrUpdateComponent(componentExists bool, ei envinfo.EnvSp
 	if err != nil {
 		return err
 	}
-
+	deployment.Spec.Replicas = pointer.Int32Ptr(1)
```

Comment:
It would be nice to have this field set when the Deployment is first created, by the devfile library. Could we propose a path to the library to add the Replicas to the `generator.DeploymentParams` structure?

Reply:
By setting it at library level, it becomes applicable to Che folks as well. I'm not sure if that's what they would want. Otherwise, I think it could be patched at library level.

IMO, we could track this separately (issue on devfile repo or a discussion in the Devfile cabal). WDYT?

Reply:
Through the `generator.DeploymentParams` structure, it could be an optional parameter, to not impact other library's users.